### ISSUE TITLE
entity vecs and model file

### ISSUE THREAD

**Issue by kanihal** [dalab/deep-ed#2]
Hi,
Could you please share the compressed version of the 'generated' folder along with trained entity vecs and model file?

The reason I'm asking is I don't have a good computing setup to train the entire thing, I would like to see how the model performs on my dataset.

**Comment by octavian-ganea** [comment]
Hi,

This cannot work as you would need to train entity vectors for all candidates in your dataset, which might be different from the candidates that appeared in the datasets that I used.  In addition, testing with a pre-trained model (via test_one_loaded_model.lua) will still use most of the resources used in training. But it should not use more than 10 GB of RAM if you do not load the entire training data to disk.

**Comment by se4u** [comment]
@octavian-ganea Just wanted to say that if will be great if you can upload the model file. Actually I personally am only interested in the weights for the `f_network` defined in the `linear_layers.lua`

```
f_network = nn.Sequential()
  :add(nn.Linear(2,opt.nn_pem_interm_size))
  :add(nn.ReLU())
  :add(nn.Linear(opt.nn_pem_interm_size,1))
```

Since the NN just takes two real inputs and gives a scalar output I wanted to see the contour plot of this part of the network to see how the two scores are being combined.

**Comment by octavian-ganea** [comment]
That's right, figure 3 is plotting that function. Unfortunately I do not have the local model, so I would have to re-run the whole training again. Not sure I will have time before the ICML/ACL deadlines. However, I hope the instructions on how to reproduce our results are easy to follow, so feel free to try it out yourself :).
